packages/charts/src/Charts/SeverityLine/SeverityLine.js

```diff
@@ -0,0 +1,73 @@
+import './severity-line.scss';
+
+import React, { useEffect, useState } from 'react';
+import { Tooltip, TooltipPosition } from '@patternfly/react-core/dist/js/components/Tooltip/index';
+
+import PropTypes from 'prop-types';
+
+const SeverityLine = ({ title, value, tooltipMessage, config }) => {
```

Comment:
You should take the rest of props in here and spread them in the div on line 38.

Reply:
I added an additional prop `chartProps` where additional props could be passed. Is that what you are intending?